benchmark_runner/grafana/perf/jsonnet/main.libsonnet

```diff
@@ -815,69 +815,10 @@ g.dashboard.new('PerfCI-Regression-Summary')
 
               ])
 
-              +elasticsearch.withQuery('_exists_:norm_ltcy AND read_message_size:(64 OR 1024 OR 8192) AND num_threads:(1) AND test_type:rr AND norm_ltcy:<1000 AND kind:$kind  AND ocp_version:$ocp_version')
+              +elasticsearch.withQuery('_exists_:norm_ltcy AND read_message_size:(64 OR 1024 OR 8192) AND num_threads:(1 OR 8) AND test_type:rr AND norm_ltcy:<1000 AND kind:$kind AND ocp_version:$ocp_version')
```

Comment:
If you're going to aggregate on all of the message sizes and number of threads, do you need any clause in there?

Reply:
OK, that answers the question.